Project.toml

```diff
@@ -1,5 +1,7 @@
 name = "Nabla"
 uuid = "49c96f43-aa6d-5a04-a506-44c7070ebe78"
+authors = ["Invenia Labs Ltd."]
```

Comment:
I believe we concluded that it should be:
`Invenia Technical Computing Corperation` 
?

Reply:
I was just following what's in the license. Where was that conclusion discussed?

Reply:
In an issue on InveniaTemplates. 
But lets wait an here from someone who actually understands our I.P. contract.
See  #dev

Reply:
Sounds good.

Reply:
I'll just remove the author section for now so as not to stall registering another version.

Reply:
`Corporation` not `Corperation` btw

Reply:
CorporalNation